internal/kafka/internal/config/kafka.go

```diff
@@ -56,6 +56,7 @@ func (c *KafkaConfig) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&c.BrowserUrl, "browser-url", c.BrowserUrl, "Browser url to kafka admin UI")
 	fs.BoolVar(&c.EnableKafkaOwnerConfig, "enable-kafka-owner-config", c.EnableKafkaOwnerConfig, "Enable configuration for setting kafka owners")
 	fs.StringVar(&c.KafkaOwnerListFile, "kafka-owner-list-file", c.KafkaOwnerListFile, "File containing list of kafka owners")
+	fs.IntVar(&c.Quota.MaxAllowedTrialInstance, "max-allowed-trial-instances", c.Quota.MaxAllowedTrialInstance, "Max allowed trial instances")
```

Comment:
If this ends up being merged, this should be MaxAllowedTrialInstances (plural)

Reply:
Also see https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pull/1226/files#r936580215. Depending on the outcome of that discussion the attribute should be renamed too